lib/src/utils/screen_util.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:flutter_device_type/flutter_device_type.dart';
+
+class ScreenUtil {
```

Comment:
We should have variables describe for these 'magic numbers'.

Reply:
done